slownik.rst

```diff
@@ -0,0 +1,178 @@
+
+IDS
+  ang. Intrusion Detection System — urządzenia poprawiające bezpieczeństwo sieci komputerowych poprzez wykrywanie ataków w czasie rzeczywistym.
+
+Implementacja
+  [za PWN] wersja programu, języka programowania lub systemu operacyjnego przystosowana do pracy na określonym typie komputera; też: przystosowanie programu, języka programowania lub systemu operacyjnego do typu komputera.
+
+Instancja
+  Maszyna lub maszyny wirtualne, w ramach których działa instalacja platformy specyficzna dla konkretnego urzędu. Przydzielona dla jednej instytucji kopia Platformy, na której będzie uruchamiany pewien zbiór Aplikacji.
+
+IPS
+  ang. Intrusion Prevention System – urządzenia sieciowe poprawiające bezpieczeństwo sieci komputerowych przez wykrywanie i blokowanie ataków w czasie rzeczywistym.
+
+JST
+  Jednostka bądź jednostki samorządu terytorialnego: gmina, powiat, województwo.
+
+MVP
+  ang. Minimum Viable Product - podstawowa wersja systemu zapewniająca podstawową funkcjonalność umożliwiającą wprowadzenie na rynek.
+
+Marketplace
+  Marketplace będzie miejscem, gdzie przedstawiciel instytucji może zarejestrować się i utworzyć swoją instancję platformy aplikacje.gov.pl. Ten komponent będzie też odpowiadał za instalowanie poszczególnych aplikacji oraz komunikację z chmurą obliczeniową w celu np. skonfigurowania nowej instancji lub zmiany parametrów istniejącej instancji.
+
+Metadane
+  Zestaw usystematyzowanych informacji, logicznie powiązanych z przesyłką, sprawą lub inną dokumentacją, ułatwiających ich wyszukiwanie, kontrolę, zrozumienie i długotrwałe przechowanie oraz zarządzanie
+
+NASK
+  NASK jest instytutem badawczym prowadzącym działalność badawczo-rozwojową w zakresie opracowywania rozwiązań zwiększających efektywność, niezawodność i bezpieczeństwo sieci teleinformatycznych oraz innych złożonych systemów sieciowych oraz działalność edukacyjną, w tym w zakresie popularyzacji idei społeczeństwa informacyjnego
+
+Kod
+  Treść programu komputerowego.
+
+Komponent
+  Część składowa systemu realizująca konkretny zbiór funkcji.
+
+KPI (kluczowy wskaźnik efektywności)
+  Finansowy i niefinansowy wskaźnik pomiaru stopnia realizacji celów. Powinien być wyrażony w liczbach, procentach itd.
+
+Keyless Signatures’ Infrastructure
+  Usługa znakowania czasem w technologii linked timestamping, działająca zgodnie z modelem przedstawionym w dokumencie `„Keyless Signatures’ Infrastructure: How to Build Global Distributed Hash-Trees”`_.
+
+Operator techniczny
+  Podmiot, do którego Beneficjent – NASK przekaże pewne aspekty zarządu technicznego Platformy.
+
+PIUiD
+  Platforma Integracyjna Usług i Danych.
+
+Pieczęć elektroniczna
+  Dane w postaci elektronicznej dodane do innych danych w postaci elektronicznej lub logicznie z nimi powiązane, aby zapewnić autentyczność pochodzenia oraz integralność powiązanych danych.
+
+Plik JAR
```

Comment:
Definicja nie jest poprawna. Używając niestandardowego ClassLoader'a da się pewnie załadować coś mniejszego.

Proponują usunąć to definicję bo "plik JAR" jest dobrze zdefiniowanym pojęciem.